### PR TITLE
feat: 🎸 update SQ v19 support for the latest changes

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

instruction and authorization IDs are no longer padded. Queries are updated accordingly

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/middleware/queries/authorizations.ts [hunk 1]
```diff
@@ -59,7 +59,6 @@ export function authorizationsQuery(
 ): QueryOptions<PaginatedQueryArgs<QueryArgs<Authorization, AuthorizationArgs>>> {
   const { args, filter } = createAuthorizationFilters(filters);
 
-  const idField = paddedIds ? 'id' : 'authId: id';
   const orderBy = paddedIds
     ? `${AuthorizationsOrderBy.CreatedBlockIdAsc}`
```
Comment (F-OBrien) [inline src/middleware/queries/authorizations.ts:63]: order by CREATED_EVENT_ID_ASC this ensure correct ordering even if multiple authorizations are emitted in a single block
```suggestion
    ? `${AuthorizationsOrderBy.CreatedEventIdAsc}`
```

File: src/middleware/queries/settlements.ts [hunk 2]
```diff
@@ -178,9 +178,6 @@ export function instructionsQuery(
 
   if (paddedIds) {
     orderBy = `[${InstructionEventsOrderBy.IdDesc}]`;
```
Comment (F-OBrien) [inline src/middleware/queries/settlements.ts:180]: when padded order by CREATED_EVENT_ID_DESC

**Review by F-OBrien — APPROVED** [review]
approved with comments

**Comment by polymath-eric** [comment]
For transparency, I manually added the CREATED_EVENT_ID field directly into the middleware types file. 

Typegen with v19 wipes all of the deprecated fields which are more numerous to add in/work around.

**Review by F-OBrien — APPROVED** [review]

**Pull request merged**